pkg/tests/support/helpers.py

```diff
@@ -182,57 +192,104 @@ def relenv(self, version):
             relenv = True
         return relenv
 
-    def get_version(self):
+    def update_process_path(self):
+        # The installer updates the path for the system, but that doesn't
+        # make it to this python session, so we need to update that
+        os.environ["PATH"] = ";".join([str(self.install_dir), os.getenv("path")])
+
+        # When the MSI installer is run from self.proc.run, it doesn't update
+        # the registry. When run from a normal command prompt it does. Until we
+        # figure that out, we will update the process path as above. This
```

Comment:
I think this can be removed now correct @twangboy ?

Reply:
No, that's still an issue.